models/demos/ttnn_resnet/tt/ttnn_functional_resnet50_large_new_conv_api.py

```diff
@@ -181,7 +181,9 @@ def run_downsample_if_req(
                     dtype=self.model_config["ACTIVATIONS_DTYPE"],
                     weights_dtype=self.model_config["WEIGHTS_DTYPE"],
                     math_fidelity=self.model_config["MATH_FIDELITY"],
-                    height_sharding=height_sharding,
+                    shard_layout=ttnn.TensorMemoryLayout.HEIGHT_SHARDED
+                    if height_sharding
+                    else ttnn.TensorMemoryLayout.BLOCK_SHARDED,
```

Comment:
can we please move this out to a separate variable calculation? it would be fine on one line, but definitely not split like that

Reply:
Fixed.